README.md

```diff
@@ -1,28 +1,157 @@
+
+```bash
+oc login -u system:admin
+oc policy add-role-to-user cluster-admin developer -n jaeger
+```
+
+#### Installing Jaeger under an existing namespace as `developer`
+
+Jaeger can also be installed as `developer` (or any other user). As the `DaemonSet` runs at the node level,
+however, your user need to have such permission. It can be achieved by running this:
+
+```bash
+oc login -u system:admin
+oc create -f daemonset-admin.yml
+oadm policy add-role-to-user daemonset-admin developer
+```
+
+Once that is ready, it's only a matter of creating the components from the template:
+```bash
+oc process -f template.yml -p NAMESPACE=myproject | oc create -f -
+```
+
+#### Post install
+
+The Docker images should be pulled automatically from Docker Hub, and all other processes should also complete
+automatically. Once everything is ready, the Jaeger Query UI can be accessed via https://jaeger-query-jaeger.127.0.0.1.nip.io/ .
+Don't forget to replace `127.0.0.1` by your OpenShift IP, shown as the result of `oc cluster up` or `oc status`.
+
+Note that it's OK to have the Query and Collector pods to be in an error state for the first minute or so. This is
+because these components attempt to connect to Cassandra right away and hard fail if they can't after N attempts.
+After a hard fail, OpenShift/Kubernetes will schedule a new pod. As Cassandra will eventually be ready, the pods
+will eventually work.
+
+This is how it looks like at different times:
+
+```bash
+$ oc get pods
+NAME                                READY     STATUS              RESTARTS   AGE
+cassandra-0                         1/1       Running             0          33s
+cassandra-1                         0/1       ContainerCreating   0          2s
+jaeger-agent-jgprc                  1/1       Running             0          33s
+jaeger-cassandra-schema-job-n1806   1/1       Running             0          33s
+jaeger-collector-1-deploy           1/1       Running             0          33s
+jaeger-collector-1-xqb7k            0/1       CrashLoopBackOff    1          28s
+jaeger-query-1-deploy               1/1       Running             0          33s
+jaeger-query-1-h089j                0/1       Error               2          28s
+jaeger-ui-1-97dk7                   1/1       Running             0          29s
+$ oc get pods
+NAME                                READY     STATUS      RESTARTS   AGE
+cassandra-0                         1/1       Running     0          1m
+cassandra-1                         0/1       Running     0          29s
+jaeger-agent-jgprc                  1/1       Running     0          1m
+jaeger-cassandra-schema-job-n1806   0/1       Completed   0          1m
+jaeger-collector-1-xqb7k            1/1       Running     2          55s
+jaeger-query-1-h089j                1/1       Running     3          55s
+```
+
+## Getting an OpenShift cluster running
```

Comment:
I would keep at least a very basic set of instructions. OpenShift is a complex software, and there are literally thousands of ways to deploy it. Mentioning `oc cluster up` and `minishift` should be short enough, while still pointing users to the "easiest" solutions for a quick start.